Packs/Sixgill-Darkfeed/IndicatorFields/indicator_cvss20score.json

```diff
@@ -0,0 +1,55 @@
+{
+	"incidentFields": [
+		{
```

Comment:
Looks like something is off with this file, why are all the data inside the "incidentFields" key? 
It should be under the json root.
@EliorKedar Please confirm

Also, please add a "fromversion" key tp all indicator fields

Reply:
Correct, should look as follow:
![image](https://user-images.githubusercontent.com/60568193/113577492-05800180-962a-11eb-96cb-f1152a14ff6e.png)